### PR TITLE
23-Feat/add-constants-Leila

### PR DESCRIPTION
**Pull request by leilafarsani**
This is a:

- [x] ✨ **New feature** - new behaviour has been implemented
- [ ] 🐛 **Bug fix** - existing behaviour has been made to behave
- [ ] ♻️ **Refactor** - the behaviour has not changed, just the implementation
- [ ] ⚙️ **Chore** - maintenance task, behaviour and implementation haven't changed

### Description

This pull request organizes server code, adds a "constants" folder, and replaces hard-coded strings with constants for better maintainability.

### Related to

Feat #23 

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have carefully reviewed my own code
- [ ] I have commented my code
- [X] I have updated any documentation
- [X] I have written a title that reflects the relevant ticket
- [X] I have written a description that says what the PR does and how to validate it
- [X] I have linked to the project board ticket (and any related PRs/issues) in the Links section
- [X] I have added a link to this PR to the ticket
- [ ] I have requested reviewers here and in my team chat channel

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { createRoot } from "react-dom/client";
 import { BrowserRouter } from "react-router-dom";
+import "./styles/index.css";
+
 
 import App from "./App";
```
Comment (Ali-Jahankah) [inline client/src/index.js:7]: We don't need these in this PR.

File: client/src/styles/index.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+*{
+    border: 0;
+    margin: 0;
+    padding: 0;
+    outline: 0;
+    box-sizing: border-box;
+    
+}
```
Comment (Ali-Jahankah) [inline client/src/styles/index.css:8]: None of these changes needed.

File: server/constants/pathConstants.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export const GET_USERS_PATH = "/get-users";
```
Comment (Ali-Jahankah) [inline server/constants/pathConstants.js:1]: Have you loged this const in the app to see if it's working or not? As Node doesn't support 'export'.
TS supports that but this starter kit may support that or may not as it's using Webpack compiler!
Test it and update me. Thanks!

**Review by Ali-Jahankah — CHANGES_REQUESTED** [review]

File: client/src/index.js (hunk 1 above)
Comment (leilafarsani) [reply client/src/index.js:7]: It's from other brach :(

File: client/src/styles/index.css (hunk 2 above)
Comment (leilafarsani) [reply client/src/styles/index.css:8]: like above :))

File: server/constants/pathConstants.js (hunk 3 above)
Comment (Ali-Jahankah) [reply server/constants/pathConstants.js:1]: Ok fine. So, files must be ".js".
log that as you are using "export const ..."; if you get undefined, then use "module.exports={name of the consts}" and it should work.
@leilafarsani

Comment (leilafarsani) [reply server/constants/pathConstants.js:1]: I meant I can btw in the other comment, sorry 🤣

Comment (Ali-Jahankah) [reply server/constants/pathConstants.js:1]: Ok fine. Please, we don't need to change any configs. This starter kit is created at the highest standard of it.
So fix it and change the PR to staging branch, then we can merge it. Thanks!

File: client/src/index.js (hunk 1 above)
Comment (leilafarsani) [reply client/src/index.js:7]: Yes TL 🫡

File: server/constants/pathConstants.js (hunk 3 above)
Comment (leilafarsani) [reply server/constants/pathConstants.js:1]: Yes TL 🫡

Comment (leilafarsani) [reply server/constants/pathConstants.js]: I cleaned the mess.
It is also tested in api.js file 
IMAGE 2023-07-27 22:36:30

**Review by Ali-Jahankah — APPROVED** [review]
Nice work!

**Pull request merged**